Tags/index.json

```diff
@@ -12,5 +12,6 @@
  "red",
  "slow",
  "trippy",
- "tron"
+ "tron",
+ "Hacienda"
```

Comment:
Needs to be lower case in here. Technically just needs to match the case of the file in the Tags directory

Reply:
fixing!